lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -8,25 +8,31 @@ const {
   ArrayPrototypeAt,
   ArrayPrototypeForEach,
   ArrayPrototypeIncludes,
-  DateNow,
+  DatePrototypeGetTime,
+  DatePrototypeToString,
   FunctionPrototypeApply,
   FunctionPrototypeBind,
+  FunctionPrototypeToString,
+  globalThis,
+  NumberIsNaN,
+  ObjectDefineProperty,
+  ObjectDefineProperties,
+  ObjectGetOwnPropertyDescriptor,
+  ObjectGetOwnPropertyDescriptors,
   Promise,
+  Symbol,
   SymbolAsyncIterator,
   SymbolDispose,
-  globalThis,
```

Comment:
globalThis should be the last (as it was)

Reply:
But isn't it alphabetically sorted?

Reply:
@aduh95 mentioned that this is the ascii order or something (I don't recall) but the lowercase ones go to the bottom